packages/plugin-list/src/ListDomObjectRenderer.ts

```diff
@@ -28,7 +28,7 @@ export class ListDomObjectRenderer extends AbstractRenderer<DomObject> {
             }
         }
         const renderedChildren = await this.engine.renderChildren(node);
-        domObject.children.push(...renderedChildren.flat());
+        domObject.children.push(...[].concat(...renderedChildren));
```

Comment:
I did not succeed making the signature of `flat` match the type of `renderedChildren`.

Reply:
Well it's because Typescript is smarter than we all are. The flat on this line is historical and not needed anymore. I expect `domObject.children.push(...renderedChildren);` to work.

Reply:
Ok I trie that

Reply:
Done, all test pass